packages/grid/_modules_/grid/hooks/features/rows/useGridEditRows.ts

```diff
@@ -82,32 +83,25 @@ export function useGridEditRows(apiRef: GridApiRef) {
 
   const setCellMode = React.useCallback(
     (id, field, mode: GridCellMode) => {
-      let hasChanged = false;
+      const isInEditMode = apiRef.current.getCellMode(id, field) === 'edit';
```

Comment:
Maybe it makes sense to add an `enum` for the `edit` and `view` modes?

Reply:
What do you mean?

Reply:
Not to use the string directly => 
```
enum CellEditMode {
  View = 'view',
  Edit = 'edit',
}
```
?

Reply:
Following this line of thinking we also need to create an enum for `asc` | `desc` and other constant values we might have. I think we can discuss if it's time to do this abstraction. We have a enum for feature mode but it's used in a lot of places, so it makes sense, but `CellEditMode` would be used in only one hook. IDK, for now TypeScript already suggests the values for us.

Reply:
Not sure, I used the same approach for the density and it is used in fewer places. 

> but CellEditMode would be used in only one hook.

And when you are calling the API and passing the mode.

Reply:
I started to add this enum but I had to change a bunch of new files that were not modified by this PR. Better to do in another PR. I still think it's a bit redundant to have the TS type and the enum, but I'm open for opinions.